File: app/src/main/java/com/playgilround/schedule/client/model/BaseResponse.java
package com.playgilround.schedule.client.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class BaseResponse<T> {
    @SerializedName("message")
    @Expose
    private T message;

    @SerializedName("status")
    @Expose
    private String code;


    public T getMessage() {
        return message;
    }

    public void setMessage(T message) {
        this.message = message;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }
}

File: app/src/main/java/com/playgilround/schedule/client/model/BaseResponse.kt
package com.playgilround.schedule.client.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class BaseResponse {
    @SerializedName("status")
    @Expose
    var code: String = ""

    @SerializedName("message")
    @Expose
    var message: String = ""

}